apps/guardian-ui/src/components/setup/screens/setConfiguration/BasicSettingsForm.tsx

```diff
@@ -38,7 +38,7 @@ export const BasicSettingsForm: React.FC<BasicSettingsFormProps> = ({
 
   const validateName = useCallback(
     (name: string) => {
-      const isValid = name.trim() !== '' && /^[a-zA-Z0-9]/.test(name);
+      const isValid = name.trim() !== 'set-config.validation-guardian-name';
```

Comment:
I'm fairly sure this isn't doing regex matching anymore. Not a TS/JS expert, but looks wrong. cc @Kodylow

Reply:
yeah this is broken, this check is doing "does the name match the hard coded value in "set-config": {"validation-guardian-name": VALUE}